docs/gettingstarted/structuring.md

```diff
@@ -14,6 +14,50 @@ Pick a unique package name. If you own a URL associated with your project, you c
 
 After the top level package (if you have one) you append a unique name for your mod, such as `examplemod`. In our case it will end up as `com.example.examplemod`.
 
+The `mcmod.info` file
+-------------------
+
+This file defines the metadata of your mod. This file lists information about your mod that users may view from the main screen of the game through the Mods button. A single info file can describe several modids. If the a mod has set `Mod.useMetadata = true`, the `mcmod.info` will override the corresponding metadata in the `@Mod` annotation. You should define every property (except `updateUrl`) so there is never an error when parsing.
```

Comment:
> Mod.useMetadata

where's that, in @Mod? clarify that

> You should define every property

i don't think this is necessary, just note in your table below what the minimal necessities are